backends/advanced/src/advanced_omi_backend/workers/obsidian_jobs.py

```
@@ -0,0 +1,107 @@
+    """Recursively count markdown files in a vault."""
+    count = 0
+    for root, dirs, files in os.walk(vault_path):
+        dirs[:] = [d for d in dirs if not d.startswith(".")]
+        for filename in files:
+            if filename.endswith(".md"):
+                count += 1
+    return count
+
+
+@async_job(redis=True, beanie=False)
+async def ingest_obsidian_vault_job(job_id: str, vault_path: str, redis_client=None) -> dict:
```

Ruff (0.14.8)

Unused function argument: `job_id`

(ARG001)